src/pages/workspaces/workspace/Analyses.js

```diff
@@ -50,36 +49,35 @@ import ExportAnalysisModal from 'src/pages/workspaces/workspace/notebooks/Export
 import { wrapWorkspace } from 'src/pages/workspaces/workspace/WorkspaceContainer'
 
 
-const analysisCardCommonStyles = listView => _.merge({ display: 'flex' },
-  listView ?
-    {
-      marginBottom: '0.5rem',
-      flexDirection: 'row',
-      alignItems: 'center'
-    } :
-    {
-      margin: '0 2.5rem 2.5rem 0',
-      height: 100,
-      width: 400,
-      flexDirection: 'column',
-      padding: 0
-    }
-)
-
 const noWrite = 'You do not have access to modify this workspace.'
 
 const sortTokens = {
-  lowerCaseName: notebook => notebook.name.toLowerCase()
+  name: notebook => notebook.name.toLowerCase()
 }
-const defaultSort = { label: 'Most Recently Updated', value: { field: 'updated', direction: 'desc' } }
-const sortOptions = [
-  defaultSort,
-  { label: 'Least Recently Updated', value: { field: 'updated', direction: 'asc' } },
-  { label: 'Alphabetical', value: { field: 'lowerCaseName', direction: 'asc' } },
-  { label: 'Reverse Alphabetical', value: { field: 'lowerCaseName', direction: 'desc' } }
-]
+const defaultSort = { label: 'Most Recently Updated', value: { field: 'lastModified', direction: 'desc' } }
+
+const analysisContextMenuSize = 16
+const centerColumnFlex = { flex: 5 }
+const endColumnFlex = { flex: '0 0 150px', display: 'flex', justifyContent: 'flex-left', whiteSpace: 'nowrap' }
 
-const AnalysisCard = ({ namespace, name, updated, metadata, toolLabel, listView, wsName, onRename, onCopy, onDelete, onExport, canWrite, currentUserHash, potentialLockers }) => {
+const AnalysisCardHeaders = Utils.memoWithName('AnalysisCardHeaders', ({ sort, onSort }) => {
```

Comment:
Does this need to be memoized?

Reply:
I'm not too sure what its doing... I copied this from the billing tab. The table does it there and this table is trying to be as close to that one for consistency

Reply:
The [memo ](https://reactjs.org/docs/react-api.html#reactmemo)is a performance optimization. I don't think we need it here and would prefer not to prematurely optimize.

The name portion allows you to give the component a [display name](https://reactjs.org/docs/higher-order-components.html#convention-wrap-the-display-name-for-easy-debugging) which can be useful for debugging, but I don't think we need it here.

![image](https://user-images.githubusercontent.com/50371603/126785274-770c7505-a7ea-44bc-80e9-f70affe92ea7.png)